content/06.model-complexity.md

```diff
@@ -1,34 +1,34 @@
+In cascade learning, multiple methods leveraging distinct underlying assumptions are used in tandem to capture stable patterns existing in the dataset [@doi:10.1109/cvpr.2001.990537; @doi:10.1007/978-3-540-75175-5_16; @doi:10.1109/icpr.2004.1334680]. 
 For example, a cascade learning approach for identifying rare disease patients from electronic health record data utilized independent steps for feature extraction (word2vec [@arxiv:1301.3781]), preliminary prediction with ensembled decision trees, and prediction refinement using data similarity metrics [@pmid:30815073]. 
-Combining these three methods resulted in better performance than other methods when implemented on the silver standard dataset in isolation.
-Other approaches like inverse class weighting, oversampling (with methods like Synthetic Minority Oversampling Technique, abbreviated SMOTE), or cascade learning approaches that adequately represent the rarer classes are also beneficial in increasing stability of predictions while accommodating for class imbalance.
+Combining these three methods resulted in better performance than other methods when implemented on a silver standard dataset in isolation.
+Other approaches like inverse class weighting, oversampling (with methods like Synthetic Minority Oversampling Technique, abbreviated SMOTE), or methods that adequately represent the rarer classes are also beneficial in increasing stability of predictions while accommodating for class imbalance.
 
-Presence of label-noise and sparsity in the data can lead to overfitting of models to the training data, such that the models show high prediction accuracy on training data but low prediction accuracy on encountering new data points. 
-Features identified as predictive by overfit models are often misrepresented in the training dataset making the model not generalizable to new data. [@doi:10.1073/pnas.1900654116]
-Regularization can not only protect ML models from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training. (Figure[@fig:2]C)
-Regularization is often used in rare variant discovery and immune cell signature discovery studies which are useful examples that also need to accommodate sparsity of data like rare diseases.
+Presence of label-noise and sparsity in the data can also lead to overfitting of models to the training data, such that the models show high prediction accuracy on training data but low prediction accuracy on encountering new data points. 
+Features identified as predictive by overfit models are often not representative of the data making the model not generalizable to new data. [@doi:10.1073/pnas.1900654116]
+In such cases, regularization can not only protect ML models from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training. (Figure[@fig:2]C)
```

Comment:
Do we define regularization before this?

Reply:
We define regularization in line 32 right after this. Let me know if that seems insufficient.. I tried a couple of different ways to define it but in case we want to include more details about regularization, we would need to go into details of shrinkage of coefficient estimates. IMO that would add more jargon than desirable.